dev/prod/webpack.config.js

```diff
@@ -151,7 +152,10 @@ module.exports = {
     new MiniCssExtractPlugin({
       filename: '[name].css'
     }),
-    new Dotenv()
+    new Dotenv(),
+    new DefinePlugin({
+      'process.env.CLIENT': JSON.stringify(process.env.CLIENT)
```

Comment:
In case Dotenv doesn't have variable we could pass it from NODE, this is used for `rushx dev-server` to start web ui and connect to server.